dashboard/rules.py

```diff
@@ -11,7 +11,7 @@ def is_admin(self, user):
     if self.context.get(user.id):
         return self.context.get(user.id)
 
-    result = user.is_superuser
```

Comment:
This was needed since all the API call after the LTI checking for if the user is super_user and that's why the user something like "You are not Part this this course 12223"